lib_comfyui/comfyui_context.py

```diff
@@ -0,0 +1,17 @@
+import sys
+import modules.scripts as scripts
+base_dir = scripts.basedir()
+
+
+class ComfyuiContext:
+    def __init__(self):
+        self.original_sys_path = list(sys.path)

```

Comment:
we may want to move this to the __enter__ method, in case sys.path is modified between the constructor and __enter__

Reply:
This shouldn't happen with the current code, but just in case the code changes later. It's a small detail.